web/portal/platform/src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,275 @@
+  name: string;
+  nif: string;
+  sipDomain: string;
+  maxCalls: number;
+}
+
+interface DashboardData {
+  admin: DashboardAdmin;
+  recentActivity: DashboardRecentActivity[];
+  brandNumber: number;
+  clientNumber: number;
+  userNumber: number;
+}
+
+interface ActiveCalls {
+  inbound: number;
+  outbound: number;
+  total: number;
+}
+
+const Dashboard = (props: DashboardProps) => {
+  const { className } = props;
+
+  const [data, setData] = useState<DashboardData | null>(null);
+  const [activeCalls, setActiveCalls] = useState<ActiveCalls | null>(null);
+  const apiGet = useStoreActions((store) => store.api.get);
+  const [, cancelToken] = useCancelToken();
+
+  useEffect(() => {
+    apiGet({
+      path: '/my/dashboard',
+      params: {},
+      cancelToken: cancelToken,
+      successCallback: async (response) => {
+        setData(response as DashboardData);
+      },
+    });
+
+    apiGet({
+      path: '/my/active_calls',
+      params: {},
+      cancelToken: cancelToken,
+      successCallback: async (response) => {
+        setActiveCalls(response as ActiveCalls);
+      },
+    });
+  }, [apiGet, cancelToken]);
+
+  if (!data || !activeCalls) {
+    return null;
+  }
+
+  const total = activeCalls.total || 0;
+  const inbound =
+    total === 0 ? '50%' : `${Math.round((activeCalls.inbound / total) * 100)}%`;
+  const outbound =
+    total === 0
+      ? '50%'
+      : `${Math.round((activeCalls.outbound / total) * 100)}%`;
+
+  return (
+    <section className={className}>
+      <div className='card welcome'>
+        <div className='card-container'>
+          <div>
+            <h3>
+              {_('Welcome to <br /> Ivoz Provider global administrator portal')}
+            </h3>
+            <p>
+              {_(
+                'Ivoz Provider is an Open Source solution by Irontec. In this portal you can add brands, brand operators, portals URL and much more.'
+              )}
+            </p>
+            <a href='/doc/en/administration_portal/platform/index.html '>
+              <LightButton>{_('Get started')}</LightButton>
+            </a>
+          </div>
+
+          <img src='assets/img/dashboard-welcome.svg' />
+        </div>
+      </div>
+      <div className='card activity'>
+        <div className='title'>{_('Operator information')}</div>
+
+        <div className='content'>
+          <div className='row'>
+            <div className='time'>Iden</div>
+            <div className='value'>{data.admin.username}</div>
+          </div>
+          <div className='row'>
+            <div className='time'>name</div>
+            <div className='value'>{data.admin.name}</div>
+          </div>
+          <div className='row'>
+            <div className='time'>lastname</div>
+            <div className='value'>{data.admin.lastname}</div>
+          </div>
+          <div className='row'>
+            <div className='time'>email</div>
+            <div className='value'>{data.admin.email}</div>
```

Comment:
These labels will require translation, using already translated strings :ok_hand: 

- Iden
- Name
- Lastname
- Email

I understand the change of Username for Iden may be related with the translation to spanish _Nombre de usuario_ beign too large and breaking the box layout.

Reply:
`I understand the change of Username for Iden`
No idea. Just followed issue definition